.lychee.toml

```diff
@@ -97,6 +97,7 @@ exclude = [
   'opensource\.org',  # 403
   'portal\.azure\.com',  # 403
   'turing\.ac\.uk',  # DDOS protection
+  'web\.archive\.org', # DDOS protection
```

Comment:
Do we have any links to this domain?

I think it was added for docs which had been removed from the Azure site.

Reply:
Ah OK. I'll remove it then.